prog/dftb+/lib_dftbplus/parser.F90

```diff
@@ -3711,6 +3716,96 @@ subroutine readSolvGB(node, geo, input)
   end subroutine readSolvGB
 
 
+  subroutine readSolvSASA(node, geo, input)
+
+    !> Node to process
+    type(fnode), pointer :: node
+
+    !> Geometry of the current system
+    type(TGeometry), intent(in) :: geo
+
+    !> Contains the input for the dispersion module on exit
+    type(TSASAInput), intent(out) :: input
+
+    type(string) :: buffer, modifier
+    type(fnode), pointer :: child, value1, value2, field, child2, dummy
+    integer :: iSp, ii, gridPoints
+    real(dp) :: conv
+    real(dp), allocatable :: vdwRadDefault(:)
+
+    if (geo%tPeriodic) then
+      call detailedError(node, "SASA model currently not available with PBCs")
+    end if
+
+    call getChildValue(node, "ProbeRad", input%probeRad, modifier=modifier, child=field)
```

Comment:
ProbeRadius might be clearer for users

Reply:
Changed it, even if it means that I have to rerender all the parameter files.